ts/src/test/Exchange/test.features.ts

```diff
@@ -6,28 +6,26 @@ async function testFeatures (exchange: Exchange, skippedProperties: object) {
     const marketTypes = [ 'spot', 'swap', 'future', 'option' ];
     const subTypes = [ 'linear', 'inverse' ];
     const features = exchange.features;
-    if (features !== undefined) {
-        const keys = Object.keys (features);
-        for (let i = 0; i < keys.length; i++) {
-            testSharedMethods.assertInArray (exchange, skippedProperties, 'features', keys, i, marketTypes);
-            const marketType = keys[i];
-            const value = features[marketType];
-            // assert (value !== undefined, 'exchange.features["' + marketType + '"] is undefined, that key should be either absent or have a value');
-            if (value === undefined) {
-                continue;
-            }
-            if (marketType === 'spot') {
-                testFeaturesInner (exchange, skippedProperties, value);
-            } else {
-                const subKeys = Object.keys (value);
-                for (let j = 0; j < subKeys.length; j++) {
-                    const subKey = subKeys[j];
-                    testSharedMethods.assertInArray (exchange, skippedProperties, 'features', subKeys, j, subTypes);
-                    const subValue = value[subKey];
-                    // sometimes it might not be available for exchange, eg. future>inverse)
-                    if (subValue !== undefined) {
-                        testFeaturesInner (exchange, skippedProperties, subValue);
-                    }
+    const keys = Object.keys (features);
```

Comment:
here also, no changes were made, only i've removed `if (features !== undefined)` check and back-indented the inner block.